moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.cpp

```diff
@@ -188,17 +185,17 @@ int PR2ArmIKSolver::cartToJntSearch(const KDL::JntArray& q_in, const KDL::Frame&
       return -1;
     q_init(free_angle_) = initial_guess + search_discretization_angle_ * count;
     if (verbose)
-      ROS_DEBUG_NAMED("pr2_arm_kinematics_plugin", "%d, %f", count, q_init(free_angle_));
-    loop_time = (ros::Time::now() - start_time).toSec();
+      RCLCPP_WARN(LOGGER_PR2_ARM_KINEMATICS_PLUGIN, "%d, %f", count, q_init(free_angle_));
+    loop_time = rclcpp::Clock().now().seconds() - start_time.seconds();
```

Comment:
We want RCL_ROS_TIME here not RCL_SYSTEM_TIME

http://docs.ros2.org/latest/api/rcl/time_8h.html#a5c734f508ce06aec7974af10ad09d071a423b9ea49c5f5119c3b9b812c01e6bc9

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/01ca1b5d2768669845acffa9324e26fa89d13752